chromatic/rainbows/actions/shift.py

```diff
@@ -1,23 +1,33 @@
 from ...imports import *
 
-__all__ = ['shift']
+__all__ = ["shift"]
 
-def shift(self, velocity=5*u.km/u.s):
+
+def shift(self, velocity=0 * u.km / u.s):
```

Comment:
I changed the default from 5 to 0 km/s, just so the default would do nothing rather than something arbitrary.